featurebyte/api/batch_feature_table.py

```diff
@@ -86,5 +100,57 @@ def describe(self, size: int = 0, seed: int = 1234) -> pd.DataFrame:
         -------
         pd.DataFrame
             Summary of the table.
+
+        Examples
+        --------
+        >>> batch_feature_table = catalog.get_batch_feature_table("batch_feature_table_name")  # doctest: +SKIP
+        >>> summary_df = batch_feature_table.describe()  # doctest: +SKIP
         """
         return super().describe(size=size, seed=seed)
+
+    def download(self, output_path: Optional[Union[str, Path]] = None) -> Path:
+        """
+        Downloads the batch feature table from the database.
+
+        Parameters
+        ----------
+        output_path: Optional[Union[str, Path]]
+            Location to save downloaded parquet file.
+
+        Returns
+        -------
+        Path
+
+        Raises
+        ------
+        FileExistsError
+            File already exists at output path.
+        RecordRetrievalException
+            Error retrieving record from API.
+
+        Examples
+        --------
+        >>> batch_feature_table = catalog.get_batch_feature_table("batch_feature_table_name")  # doctest: +SKIP
+        >>> downloaded_path = batch_feature_table.download(output_path="path/to/download")  # doctest: +SKIP
+
+        # noqa: DAR402
```

Comment:
Should we remove this line?

Reply:
this is to skip some linting around having unnecessary raises. if we remove this, i'll have to remove the `Raises` section, which is technically part of the docstring here too.

Reply:
Good to know this. Just to double check in case it is left unintended.